integration-cli/docker_cli_run_test.go

```diff
@@ -144,18 +144,17 @@ func TestRunLeakyFileDescriptors(t *testing.T) {
 	logDone("run - check file descriptor leakage")
 }
 
-// it should be possible to ping Google DNS resolver
+// it should be possible to lookup Google DNS
 // this will fail when Internet access is unavailable
-func TestRunPingGoogle(t *testing.T) {
+func TestRunLookupGoogleDns(t *testing.T) {
```

Comment:
This is a really minor issue, but `Dns` should be `DNS` here.

My source is that the Docker [coding style](https://github.com/docker/docker/blob/master/CONTRIBUTING.md#coding-style) document says:

> 3\. All code should follow the guidelines covered in [Effective Go](http://golang.org/doc/effective_go.html) and [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments).

Which has a section on initialisms at https://github.com/golang/go/wiki/CodeReviewComments#initialisms:

> Words in names that are initialisms or acronyms (e.g. "URL" or "NATO") have a consistent case. For example, "URL" should appear as "URL" or "url" (as in "urlPony", or "URLPony"), never as "Url". Here's an example: ServeHTTP not ServeHttp.

Though I can see this file has multiple other instances of `Dns` instead of `DNS`, so this is probably a wide spread issue that is best addressed at scale.